kpatch-build/kpatch-cc

```diff
@@ -61,7 +61,7 @@ if [[ "$TOOLCHAINCMD" =~ "gcc" ]] ; then
 		fi
 		shift
 	done
-elif [[ "$TOOLCHAINCMD" = "ld" ]] ; then
+elif [[ "$TOOLCHAINCMD" = "ld" || "$TOOLCHAINCMD" = "ld.lld" ]] ; then
```

Comment:
can we do this matching instead?
[[ "$TOOLCHAINCMD" =~ .*ld$ ]]

Reply:
This sounds a bit too broad. Maybe something like `^(.*-)?ld$` and the same for ld.lld? And probably the same regex for compiler matching while we are at it.

Reply:
right, for example `gold` is a GNU linker in binutils (that the kernel doesn't support) that would match `*ld`.